pilot/pkg/networking/plugin/registry/registry.go

```diff
@@ -30,6 +31,7 @@ var availablePlugins = map[string]plugin.Plugin{
 	plugin.Authn:            authn.NewPlugin(),
 	plugin.Authz:            authz.NewPlugin(authz.Local),
 	plugin.MetadataExchange: metadataexchange.NewPlugin(),
+	plugin.PrometheusStats:  stats.NewPlugin(),
```

Comment:
Do you think it make more sense to have a single plugin for all the telemetry stuff of a bunch of "micro plugins"? I don't have a strong opinion, curious what you think?

Reply:
> Do you think it make more sense to have a single plugin for all the telemetry stuff of a bunch of "micro plugins"? I don't have a strong opinion, curious what you think?

I guess we could, I need to verify.